sdk/test-utils/perfstress/test/noop.spec.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest } from "../src";
+
+/**
+ * Should test the raw performance impact of the PerfStress framework for synchronous tests.
+ */
+export class NoOpSync extends PerfStressTest<string> {
+  public options = {};
+  run(): void {}
+}
+
+/**
+ * Should test the raw performance impact of the PerfStress framework for asynchronous tests.
+ */
+export class NoOpAsync extends PerfStressTest<string> {
```

Comment:
This can't be done because async tests can't run synchronously. Let's explore what we mean, perhaps there's a way to accommodate for what you describe.

Reply:
Though currently, the command line defines whether to run sync tests or async tests, yet indeed it doesn't allow the same test to run in either mode.

Reply:
I can merge the classes and just use Run or RunAsync depending on the CLI parameters.

Reply:
Oh! Turns out I just hadn't merged the tests. The class is already just one that allows both, and the --sync command picks the method.